### PR TITLE
Added exception message class for handle exceptions

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bug-id: Tendrl/commons#168
tendrl-spec: logging_implementation

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @brainfunked @nthomas-redhat @r0h4n please review this

**Comment by GowthamShanmugam** [comment]
@shtripat review please

**Comment by GowthamShanmugam** [comment]
@shtripat sure

**Comment by GowthamShanmugam** [comment]
sample code for use exception class:
 try:
            raise ValueError
        except ValueError as ex:
            Event(ExceptionMessage(priority="error", publisher="node-agent", payload={"message": "exception message", "exception": ex}))

**Comment by GowthamShanmugam** [comment]
@shtripat i have tested this with built-in exception and user defined exception, it is working fine

File: tendrl/commons/message.py [hunk 1]
```diff
@@ -108,6 +150,8 @@ def serialize_message(obj):
     elif isinstance(obj, datetime.datetime):
         serial = obj.isoformat()
         return serial
+    elif isinstance(obj, Exception):
```
Comment (shtripat) [inline tendrl/commons/message.py]: What is this meant for?

Comment (GowthamShanmugam) [inline tendrl/commons/message.py]: @shtripat we are string {"exception": ex} in payload so ex is an exception object, problem is object in json gives serialization problem, so at the dumping an object this code will called automatically and check any object is there. if any object of Exception type then it is dumped as yaml. this only for serialization problem

Comment (shtripat) [inline tendrl/commons/message.py]: ok

**Comment by GowthamShanmugam** [comment]
@brainfunked and @r0h4n  review please

**Comment by GowthamShanmugam** [comment]
@r0h4n please review

**Comment by GowthamShanmugam** [comment]
@brainfunked ok :)

**Comment by GowthamShanmugam** [comment]
@r0h4n yes i tested it is works fine

**Comment by r0h4n** [comment]
Please send this PR on develop branch @GowthamShanmugam

**Pull request merged**